src/features/crops/components/Field.tsx

```diff
@@ -42,7 +44,33 @@ export const Field: React.FC<Props> = ({
     setShowPopover(false);
   };
 
+  let timeoutId: null | ReturnType<typeof setTimeout> = null
+
+  const handleMouseHover = () => {
```

Comment:
We may be able to take a simpler approach here using the field data. In this case could we check that the field was recently planted. @tiffanydys 

I.e.

```
const handleMouseHover = () => {
    // Field was just planted
    if (field.plantedAt < Date.now() - 1000) {
        return;
    }
    setShowCropDetails(true);
}
```
Setting up listeners can get a bit dangerous like this as we need to manage them and ensure we clean them up in the case the component unmounts.

Reply:
Hi @adamhannigan,
I think you meant `now - field.plantedAt < 1000`
I've followed your approach on my recent update, it's much cleaner now and easier to understand 😄 

I've updated the code and the video evidence. Please test it out when you got time. ✌🏻
Thanks for your guidance ❤️😄